pkg/templates/charts/toggle/cluster-lifecycle/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright Contributors to the Open Cluster Management project
+
+###############################################################################
+# Licensed Materials - Property of IBM
+# (C) Copyright IBM Corporation 2020 All Rights Reserved
+# US Government Users Restricted Rights - Use, duplication or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
```

Comment:
I think the IBM license is not needed anymore.

Reply:
Removed